doc/cygnus-ngsi/flume_extensions_catalogue/README.md

```diff
@@ -16,6 +16,7 @@
         * [Last Data Functionality](./last_data_function.md)
     * [NGSIPostGISSink](./ngsi_postgis_sink.md)
         * [Last Data Functionality](./last_data_function.md)
+    * [NGSIOracleSQLSink](./ngsi_oracle_sink.md)
```

Comment:
No "Last Data Functionality" sub-bullet? Does that means that lastdata is not yet implemented in the OracleSink?

Reply:
In the ngsi_oracle_sink.md file I see:

```
| last\_data\_mode | no | insert | <i>insert</i>  to set last data mode. Check this [link](./last_data_function.md) for more details. In oracle sink <i>both</i> and <i>upsert</i> are not avaiable |
```

so maybe it's just a typo here.

Reply:
Clarified at skype. NTC